lib/pengine/bundle.c

```diff
@@ -1520,6 +1520,7 @@ pe__bundle_xml(pcmk__output_t *out, va_list args)
         }
 
         if (!printed_header) {
+            const char *unique = pcmk__flag_text(rsc->flags, pcmk_rsc_unique);
```

Comment:
the use of this got attached to the next commit

Reply:
Thanks

Reply:
Thanks